client/components/App.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+import scss from '../assets/app.scss';
+import AppMenu from './menu/Index';
+
+class App extends React.Component {
```

Comment:
That's great. Will fix all that. And creating a container component for each, separating logic from presentation